.changeset/ninety-peaches-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roboplay/robo.js': patch
+---
+
+feat(robo/flashcore): support for namespace arrays
```

Comment:
You don't need the `robo/` scope here because changesets are automatically attributed to the correct package. :3